examples/README.md

```diff
@@ -71,176 +71,177 @@ git checkout v0.4.0
 <!-- markdownlint-disable-next-line MD026 -->
 ## Hello, World!
 
-Example | Main | Description
---- | --- | ---
-`hello_world` | [`hello_world.rs`](./hello_world.rs) | Runs a minimal example that outputs "hello world"
+| Example       | Main                                 | Description                                       |
```

Comment:
All of this formatting stuff should probably be its own PR?

Reply:
Probably. It was just a nuisance to do as a separate PR due to format-on-save settings in my IDE. I can split it though if you think it's worth it :/

Reply:
those tables should not be formatted that way actually, otherwise as soon as someone adds a new example with a longer column, you have to change all the lines which break the diff

Reply:
Sad. Okay, I'll fix it.